### PR TITLE
data:olatocode

### PR DESCRIPTION
**Pull request by olatocode**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3143"></a>

### CODE REVIEW

**Review by Rohith-JN — CHANGES_REQUESTED** [review]
Missing a `d` in `https://github.com/olatocoe.png`

**Review by Rohith-JN — CHANGES_REQUESTED** [review]
[olatocoed](https://github.com/olatocoed.png) doesn't exist, change it to `https://github.com/olatocode.png`

File: data/olatocode.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Tobi Awosola",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Fullstack Developer | Open Source | Javascript | Node.js | MongoDB | React.js",
+  "avatar": "https://github.com/olatocoed.png",
```
Comment (EmmaDawsonDev) [inline data/olatocode.json:6]:
```suggestion
  "avatar": "https://github.com/olatocode.png",
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
There's just a small typo to fix in your avatar link, then this is good to merge.

**Pull request merged**